modules/behavioural_qc/php/models/conflictsdto.class.inc

```diff
@@ -0,0 +1,65 @@
+<?php declare(strict_types=1);
+/**
+ * PHP version 7
+ *
+ * @category Behavioural
+ * @package  Loris
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ *           Alizée Wickenheiser <alizee.wickenheiser@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris-Trunk
+ */
+
+namespace LORIS\behavioural_qc\Models;
+
+/**
+ * A ProfileDTO represents an profile in the data team helper module.
```

Comment:
All the same comments from BehaviouralDTO apply here. Also, there is no data team helper module.